test/controllers/projects_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class ProjectsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @project = projects(:one)
+  end
+
+  test "should get index" do
+    get projects_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_project_url 'html'
+    assert_response :success
+  end
+
+  test "should create project" do
+    category = project_categories(:one)
+    assert_difference('Project.count') do
+      post projects_url, params: { project: { name: 'test', category_id: category.id } }.to_json, headers: { 'Content-Type' => 'application/json' }
```

Comment:
Ah ha! I found a cleaner way to do this! I'll update and then this is good to go.

http://api.rubyonrails.org/classes/ActionDispatch/IntegrationTest.html